actions/install_provisioning_profile.rb

```diff
@@ -1,19 +1,20 @@
 module Fastlane
   module Actions
     module SharedValues
-      INSTALL_PROVISIONING_PROFILE_CUSTOM_VALUE = :INSTALL_PROVISIONING_PROFILE_CUSTOM_VALUE
     end
 
     class InstallProvisioningProfileAction < Action
```

Comment:
Part of me thinks this name should be pluralized, given it now installs multiple profiles, but the other part of me thinks it shouldn't because that would break any client using it. 

I am torn. 
![](https://media.giphy.com/media/quQ5pJEjeZwTm/giphy.gif)

Reply:
No you're totally correct, Natalie

Reply:
Updated